src/main/java/org/sonarlint/cli/report/HtmlReport.java

```diff
@@ -80,10 +80,15 @@ private void copyRuleHtmlDescriptions(Function<String, RuleDetails> ruleDescript
       copyDependency(target, "rule.css");
       for (String ruleKey : ruleKeys) {
         RuleDetails ruleDetails = ruleDescriptionProducer.apply(ruleKey);
+        String htmlDescription = ruleDetails.getHtmlDescription();
+        String extendedDescription = ruleDetails.getExtendedDescription();
+        if (extendedDescription != null && !extendedDescription.isEmpty()) {
```

Comment:
Well spotted, I meant to fix that fishy condition but forgot to fix it in the end. No it cannot be null at runtime, I only had an NPE in a test case, because of a mock. I adjusted the test case and dropped the fishy condition.